src/main/kotlin/net/lumalyte/lg/infrastructure/placeholders/LumaGuildsExpansion.kt

```
@@ -71,6 +134,33 @@ class LumaGuildsExpansion : PlaceholderExpansion(), KoinComponent {
             "guild_mode" -> guild.mode.toString()
             "has_guild" -> "true"
 
+            // Age and progression detail
+            "guild_age_days" -> Duration.between(guild.createdAt, Instant.now()).toDays().toString()
+            "guild_xp_total" -> safeProgression(guildId)?.totalExperience?.toString() ?: "0"
+            "guild_xp_this_level" -> safeProgression(guildId)?.experienceThisLevel?.toString() ?: "0"
+            "guild_xp_next_level" -> safeProgression(guildId)?.experienceForNextLevel?.toString() ?: "0"
+            "guild_xp_to_next" -> safeProgression(guildId)?.experienceToNextLevel?.toString() ?: "0"
+            "guild_xp_progress_pct" -> {
+                val p = safeProgression(guildId)
+                if (p != null) String.format("%.1f", p.levelProgress * 100.0) else "0.0"
```

detekt (1.23.8)

[warning] String.format("%.1f", p.levelProgress * 100.0) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)